File: lab9/Zad1.java
import java.lang.*;

public class Zad1 {
  public static void main(String[] args) {
    long start;

    start = System.nanoTime();
    int[] tab1 = new int[10000];
    long t1 = System.nanoTime() - start;
    System.out.println(t1 + "ns");

    start = System.nanoTime();
    int[] tab2 = new int[100000];
    long t2 = System.nanoTime() - start;
    System.out.println(t2 + "ns");

    start = System.nanoTime();
    int[] tab3 = new int[1000000];
    long t3 = System.nanoTime() - start;
    System.out.println(t3 + "ns");
  }
}

File: lab9/zad1.js
// jshint esversion: 6
const now = require('performance-now');
let start;

start = now();
let tab1 = new Array(10000);
let t1 = now() - start;
console.log('Execution time: %dms', t1);

start = now();
let tab2 = new Array(100000);
let t2 = now() - start;
console.log('Execution time: %dms', t1);

start = now();
let tab3 = new Array(10000000);
let t3 = now() - start;
console.log('Execution time: %dms', t1);
